cmd/kube-proxy/app/server.go

```diff
@@ -21,6 +21,7 @@ package app
 import (
 	goflag "flag"
 	"fmt"
+	"k8s.io/apimachinery/pkg/util/validation/field"
```

Comment:
sorted incorrectly; "namespaced" imports are kept separate from standard library imports

(likewise in other files)

Reply:
Updated.